build_schema.py

```diff
@@ -106,7 +111,41 @@ def loadYAML(location):
         if "x-kubernetes-group-version-kind" in type_def:
             for kube_ext in type_def["x-kubernetes-group-version-kind"]:
                 if "properties" not in type_def:
+                    invalid_specs_groups.append(type_def["x-kubernetes-group-version-kind"][0]["group"])
```

Comment:
Why do you loop over all `type_def["x-kubernetes-group-version-kind"]`'s members, when you always use the first element only?